docs/GUIDE.md

```diff
@@ -0,0 +1,229 @@
+# A Warrior's Guide to Thunderdome
+
+Thunderdome is a fun way to perform agile scrum planning. See Wikipedia.
+
+As a new warrior in this realm, let this be your guide to Battles. First, we need to know who you are.
+
+## Register (optional)
+
+Create a new account, or join as guest.
+
+Having an account lets you save your battles and more.
+
+- Name
+This will be visible to others.
+- Email (for account)
+- Password (for account)
+Use a strong password. Type it again to confirm
+
+You will receive an email to confirm your new account.
+
+## Login
+
+Use the email/password you created when registering.
+
+- Email
+- Password
+
+_OIDC Providers coming soon._
+
+### Password Retrieval
+
+Forgot your password? Thunderdome can send a link to your email.
```

Comment:
`can send a password reset link to your email` not to be confused with login by email which is a feature we may implement in the future.

Reply:
```suggestion
Forgot your password? Thunderdome can send a password reset link to your email.
```